sql/mob_droplist.sql

```diff
@@ -17137,6 +17202,19 @@ INSERT INTO `mob_droplist` VALUES (3209,0,0,1000,14168,1000); -- (Sabotender Bai
 INSERT INTO `mob_droplist` VALUES (3209,0,0,1000,916,140); -- (Sabotender Bailarin) Cactaur Needle
```

Comment:
Looks like this file isn't based on LSB.  You've got additional entries here for Bailarin, and also missing a couple hundred entries near the end of the file.

Reply:
hum strange I didnt select that file for this pr but I tryied to base it on my other pr I must have mess something up.